src/commands/send_event.rs

```diff
@@ -159,14 +159,20 @@ pub fn make_command(command: Command) -> Command {
         )
 }
 
-fn send_raw_event(event: Event<'static>, dsn: Dsn) -> Uuid {
-    debug!("{:?}", event);
-    with_sentry_client(dsn, |c| c.capture_event(event, None))
+pub(super) fn send_raw_event(event: Event<'static>) -> Result<Uuid> {
```

Comment:
since this is not taking a `Dsn` anymore, it just picks that up from the environment?